group_vars/all.yml

```diff
@@ -728,6 +731,15 @@ firefly_available_externally: "false"
 firefly_data_directory: "{{ docker_home }}/firefly"
 firefly_port: "8066"
 
+###
+### grocy
+###
+grocy_available_externally: "false"
+grocy_data_directory: "{{ docker_home }}/grocy"
+grocy_user_id: "1000"
```

Comment:
Shouldn't this be `0`?

Reply:
From what I remember this is one of those containers that you can't run as root due to the web server.